res/css/views/rooms/_EventTile.scss

```diff
@@ -674,3 +674,60 @@ $hover-select-border: 4px;
         margin-right: 0;
     }
 }
+
+
+.mx_ThreadInfo:hover {
+    cursor: pointer;
+}
+
+.mx_ThreadView {
+
```

Comment:
extreme nit: remove excess blank line, for consistency.

Reply:
There are unfortunately no lint rules for this one, the closest would be [`no-empty-first-line`](https://stylelint.io/user-guide/rules/list/no-empty-first-line), and it doesn't do what we want